erpc_python/erpc/transport.py

```diff
@@ -105,6 +107,26 @@ def _base_receive(self, count):
 class ConnectionClosed(Exception):
     pass
 
+class RpmsgTransport(Transport):
```

Comment:
Make `RpmsgTransport` a subclass of `FramedTransport`. This will allow you to avoid having to do an arbitrary 2048-byte receive. Instead, the `FramedTransport` will do fixed length receives of the 4-byte frame header followed by the known message length.